packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/imagehelper/ResourceDrawableIdHelper.kt

```diff
@@ -67,5 +67,15 @@ public class ResourceDrawableIdHelper private constructor() {
     @JvmStatic
     public val instance: ResourceDrawableIdHelper
       get() = resourceDrawableIdHelper
+
+    /**
+     * We're just re-adding this to reduce a breaking change for libraries in React Native 0.75.
+     *
+     * @deprecated Use instance instead
+     */
+    @Deprecated("Use instance .instead, this is ", ReplaceWith("instance"))
```

Comment:
```suggestion
    @Deprecated("Use .instance instead, this is ", ReplaceWith("instance"))
```

Reply:
Thanks for spotting it, I've fixed it in the upcoming revision 👍